File: android/app/src/androidTest/java/com/example/insta_story_demo/MainActivityTest.java
package com.example.insta_story_demo;

import androidx.test.rule.ActivityTestRule;

import com.example.insta_stotry_demo.MainActivity;

import org.junit.Rule;
import org.junit.runner.RunWith;

import dev.flutter.plugins.integration_test.FlutterTestRunner;

@RunWith(FlutterTestRunner.class)
public class MainActivityTest {
    @Rule
    public ActivityTestRule<MainActivity> rule = new ActivityTestRule<>(MainActivity.class, true, false);
}

File: android/app/src/androidTest/kotlin/com.example.insta_story_demo/MainActivityTest.kt
package com.example.insta_story_demo

import androidx.test.rule.ActivityTestRule
import com.example.insta_stotry_demo.MainActivity
import dev.flutter.plugins.integration_test.FlutterTestRunner
import org.junit.Rule
import org.junit.runner.RunWith


@RunWith(FlutterTestRunner::class)
class MainActivityTest {
    @Rule
    val rule = ActivityTestRule(
        MainActivity::class.java, true, false
    )
}
